src/Agent.Worker/AgentPluginManager.cs

```diff
@@ -133,6 +137,8 @@ public async Task RunPluginTaskAsync(IExecutionContext context, string plugin, D
                 // Execute the process. Exit code 0 should always be returned.
                 // A non-zero exit code indicates infrastructural failure.
                 // Task failure should be communicated over STDOUT using ## commands.
+                // Agent.PluginHost's arguments
+                string arguments = $"task \"{plugin}\"";
```

Comment:
Any reason this was moved? Not really a problem, but it does make the comments a little confusing.

Reply:
I think I just moved it because it was defined so far away from where it was being used.

Reply:
Oh, I guess I should have put an extra line between that existing comment block and the relocated, unrelated, unnecessary comment that went along with the variable declaration.